### ISSUE TITLE
error "list object has no element 1" while checking supervisor tasks are present

### ISSUE THREAD

**Issue by davisagli** [plone/ansible.plone_server#53]
I'm getting this error when running the Plone ansible-playbook, on both the initial and a subsequent run:

```
TASK [plone.plone_server : Supervisor zeoclient tasks are present] *************
fatal: [redacted]: FAILED! => {"failed": true, "msg": "ERROR! ERROR! ERROR! list object has no element 1"}
```

- ansible-playbook: fresh clone from today
- ansible version 2.0.0.2
- I set plone_client_count: 4; not sure if that's relevant

Any idea what's wrong?

**Comment by davisagli** [comment]
I reverted to my old version of ansible (1.7.2 and tried again, and it ran to completion. So maybe a problem related to ansible 2.x? I got into this situation because the docs say "Install a current version of Ansible," so I assumed 2.x would be correct.

**Comment by fulv** [comment]
I got this, too, but didn't have time to submit a PR.  There is something
wrong with the loop that was added to be able to add multiple sites.
AFAICT it fails when you have only one.

**Comment by fulv** [comment]
Sorry, I was going from (poor) memory.  The issue is the `with_sequence:`.  I had to comment out both `Supervisor zeoclient tasks are present` and `Supervisor zeoclients tasks are restarted` in `plone.plone_server/tasks/main.yml` to get it to run successfully.

**Comment by smcmahon** [comment]
Reproduced with Ansible 2.0.2 and our standard tests. No problem with Ansible <  2.

**Comment by smcmahon** [comment]
Looks like ansible 2 is evaluating the count expression in a with_sequence operation even when it's being skipped.

Workaround if you want to use ansible 2 while we figure this out: delete the unused plone_server role calls from playbook.yml.

**Comment by smcmahon** [comment]
Problem was actually in ansible-playbook. Fixed there by commit 3b4efb8

**Comment by davisagli** [comment]
Thanks!
